engine/src/main/java/com/datatorrent/stram/engine/AbstractReservoir.java

```diff
@@ -47,7 +49,7 @@
 {
   private static final Logger logger = LoggerFactory.getLogger(AbstractReservoir.class);
   static final String reservoirClassNameProperty = "com.datatorrent.stram.engine.Reservoir";
-  private static final int USE_SPSC_CAPACITY = 8 * 1024;
+  private static final int USE_SPSC_CAPACITY = 64 * 1024;
```

Comment:
As SpscArrayBlockingQueueReservoir provides larger throughtput compared to ArrayBlockingQueueReservoir, it exceeds throughput of SpscReservoir for larger queue capacity.